app/src/Application/DataProvider.cpp

```diff
@@ -0,0 +1,23 @@
+#include "DataProvider.h"
+
+using namespace Status::Application;
+
+namespace StatusGo = Status::StatusGo;
+
+DataProvider::DataProvider()
+    : QObject(nullptr)
+{
+}
+
+StatusGo::Settings::SettingsDto DataProvider::getSettings() const
+{
+    try {
```

Comment:
@MishkaRogachev `StatusQtGo` is just a gate to call `status-go` endpoints, since procs from `StatusQtGo` may throw an exception, also parsing json may throw an exception and since we don't have that in `StatusQtGo` (we actually should not handle them there, cause in some cases we will want to notify a user about what's happening behind) and since we don't have shared service layer, we have to handle them in the place where we're calling them.